src/lib.rs

```diff
@@ -2016,7 +2052,6 @@ macro_rules! select {
 /// Useful if you want to extract elements from a token in a zero-copy manner.
 ///
 /// `select_ref` requires that the parser input implements [`BorrowInput`].
-// TODO: Remove this, somehow unify with `select`?
```

Comment:
Since there doesn't seem to be a better way after discussion, figured this TODO can go away

Reply:
Yep, that's reasonable